### PR TITLE
feat: created landing page , added routes

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
Landing Page
I have added the landing page . So ignore other files. Focus only on the Landingpage.js (in pages ) and Header.jsx & Footer.jsx in component folder in src.

Files to check
src/components/header/header.jsx
src/components/footer/footer.jsx
src/pages/landingpage/Landingpage.js
PS - Ignore other files. . 

What not to check
Responsiveness
Navbar functionalities

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (arjun-computer-geek) [inline public/index.html:7]: Add title please

File: src/asset/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import img1 from './shoe-main.jpg'
+import img2 from "./men-shoe.jpg"
+import img3 from "./women-shoe.jfif"
+import img4 from "./kid-shoe.jfif"
+import img5 from "./trending-shoe-1.jpg"
+import img6 from "./trending-shoe-2.jpg"
+import img7 from "./trending-shoe-3.jpg"
+import wishlistimg1 from './card-img(2).jfif'
+import cartimg1 from "./nike-running.jfif"
```
Comment (arjun-computer-geek) [inline src/asset/index.js:9]: This is the good way of using images. I liked it.

File: src/components/footer/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+export const Footer = () => {
+    return(
+
+    <footer class="ecomm-footer">
+<div class="footer-container">
+    <div class="footer-row">
+        <div class="footer-col">
+            <p class="footer-col-head">Customer Service</p>
+            <ul>
+                <li><a href="#">My Account</a></li>
```
Comment (arjun-computer-geek) [inline src/components/footer/Footer.jsx:13]: You can use Link instead of a tag

File: src/components/footer/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                    class="fab fa-facebook-square"></i></a></li>
+                <li><a href="https://www.linkedin.com/in/shubham-kumar-6625b8217/" target="_blank" class="social-link"><i
+                    class="fab fa-linkedin"></i></a></li>
+                  
+                    
+            </ul>
+        </div>
+    </div>
+</div>
+<div class="copyright-container">
```
Comment (arjun-computer-geek) [inline src/components/footer/Footer.jsx:69]: use className instead of class

**Review by arjun-computer-geek — APPROVED** [review]
Do suggested changes. Else look good. I like the way of using import and export. This is good way of doing this.

File: public/index.html (hunk 1 above)
Comment (Shubhamkr2610) [reply public/index.html:7]: okay...I will do it

File: src/components/footer/Footer.jsx (hunk 3 above)
Comment (Shubhamkr2610) [reply src/components/footer/Footer.jsx:13]: I will update it

File: src/components/footer/Footer.jsx (hunk 4 above)
Comment (Shubhamkr2610) [reply src/components/footer/Footer.jsx:69]: I am changing

**Review by Muzammil-khan710 — APPROVED** [review]
Code is quite readable.
UI is clean.
Good to merge after updating the above requested changes

**Review by Rockyraven — APPROVED** [review]
your all code looks goods to me
the way of importing image is goods tom . I like it

**Pull request merged**